src/highdicom/content.py

```diff
@@ -1478,133 +1525,603 @@ def __init__(
+                    '"modality_lut" is specified.'
+                )
+            if rescale_type is not None:
+                raise TypeError(
+                    'Argument "rescale_type" must not be specified when '
+                    '"modality_lut" is specified.'
+                )
+            if not isinstance(modality_lut, ModalityLUT):
+                raise TypeError(
+                    'Argument "modality_lut" must be of type '
+                    'highdicom.ModalityLUT.'
+                )
+            self.ModalityLUTSequence = [modality_lut]
+        else:
+            if rescale_intercept is None:
+                raise TypeError(
+                    'Argument "rescale_intercept" must be specified when '
+                    '"modality_lut" is not specified.'
+                )
+            if rescale_slope is None:
+                raise TypeError(
+                    'Argument "rescale_slope" must be specified when '
+                    '"modality_lut" is not specified.'
+                )
+            if rescale_type is None:
+                raise TypeError(
+                    'Argument "rescale_type" must be specified when '
+                    '"modality_lut" is not specified.'
+                )
+            self.RescaleIntercept = format_number_as_ds(rescale_intercept)
+            self.RescaleSlope = format_number_as_ds(rescale_slope)
+            if isinstance(rescale_type, RescaleTypeValues):
+                self.RescaleType = rescale_type.value
+            else:
+                _check_long_string(rescale_type)
+                self.RescaleType = rescale_type
+
+
+class PresentationLUT(LUT):
+
+    """Dataset describing an item of the Presentation LUT Sequence."""
+
+    def __init__(
+        self,
+        first_mapped_value: int,
+        lut_data: np.ndarray,
+        lut_explanation: Optional[str] = None
+    ):
+        """
+
+        Parameters
+        ----------
+        first_mapped_value: int
+            Pixel value that will be mapped to the first value in the
+            lookup-table.
+        lut_data: numpy.ndarray
+            Lookup table data. Must be of type uint8 or uint16.
+        lut_explanation: Union[str, None], optional
+            Free-form text explanation of the meaning of the LUT.
+
+        """
+        super().__init__(
+            first_mapped_value=first_mapped_value,
+            lut_data=lut_data,
+            lut_explanation=lut_explanation
+        )
+
+
+class PresentationLUTTransformation(Dataset):
+
+    """Dataset describing the Presentation LUT Transformation as part of the
+    Pixel Transformation Sequence to transform polarity pixel values into
+    device-indendent presentation values (P-Values).
+
+    """
+
+    def __init__(
+        self,
+        presentation_lut_shape: Optional[
+            Union[PresentationLUTShapeValues, str]
+        ] = None,
+        presentation_luts: Optional[Sequence[PresentationLUT]] = None,
```

Comment:
At least for GSPS, this sequence must be of length 1 (https://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.11.6.html#table_C.11.6-1)

Suggest changing the type of this parameter to `Optional[PresentationLUT]`

Reply:
Addressed via fa147db72fd6598b2d9ee8e1f4e97cdbceff1362